biothings_explorer/call_apis_v2/query.py

```diff
@@ -0,0 +1,95 @@
+import logging
+
+import requests
+
+from .helpers import yaml_2_json
+from .metakg.parser import MetaKGParser
+from .parser import format_response
+
+
+logger = logging.getLogger(__name__)
+
+
+class SmartAPI:
+    def __init__(self, url, id=None):
+        self.url = url
+        self._id = id
+
+    @property
+    def metadata(self):
+        if not hasattr(self, "_metadata"):
+            resp = requests.get(self.url)
+            resp.raise_for_status()
+            self._metadata = yaml_2_json(resp.text)
```

Comment:
I think no, because I didn't know that before, and only tested this url: `https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/mygene.info/openapi_full.yml`

Can you provide me another URL for json format? so I can check and update the code
@newgene

Reply:
I don't have an JSON examples right now, as we always use YAML for our API metadata. But in theory, metadata can be in JSON as well. You may export a JSON version from the SmartAPI Editor using the yaml file you have.

Reply:
I already added support for JSON-based API metadata, please check :)
@newgene